rebar.config.script

```diff
@@ -13,7 +13,10 @@ NoHaveMapsOpts = try maps:size(maps:new()) of
     %% needed with rebar for windows/cygwin
     "sh build/mk_version_hrl"
     "    < include/gpb_version.hrl.in"
-    "    > include/gpb_version.hrl "}
+    "    > include/gpb_version.hrl "},
+   {compile,
+    %% way of invoking shell script: see above
+    "sh build/compile_descriptor"}
```

Comment:
Aah. Well damn.. hm. So with rebar3 it does not compile to `./ebin` but to `_build/default/lib/gpb/ebin`. With the use of `pre` the hook would write to `./ebin/` which gets copied to `_build/default/lib/gpb/ebin` when things are compiled.

I guess I'll have to add a check to have it compile to `_build` if it exists and `./ebin` if it exists.